publication-commons/src/main/resources/features/update_publication.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Update an existing Publication
+
+  Scenario: An Anonymous User wants to update a Publication
+    Given an Anonymous User wants to update a Publication
+    When they set the Accept header to "application/json"
+    And they set the request body to UpdatePublicationRequest
+    And they request PUT /publication/{identifier}
+    Then they receive a response with status code 401
+    And they see that the response Content-Type header is "application/problem+json"
+    And they see that the response body is a problem.json object
+    And they see the response body has a field "title" with the value "Unauthorized"
+    And they see the response body has a field "status" with the value "401"
+
+  Scenario: A User wants to update a Publication with invalid metadata
+    Given an Anonymous User wants to update a Publication with invalid metadata
```

Comment:
kan en `Anonymous User` oppdatere en publikasjon?
samme for scenarioene under

Reply:
Alvorlig copy-paste feil